apps/tangle-dapp/constants/walletAndChain.ts

```diff
@@ -0,0 +1,5 @@
+/** Key for source chain query params */
+export const SOURCE_CHAIN_KEY = 'source';
+
+/** Key for destination chain query params */
+export const DEST_CHAIN_KEY = 'dest';
```

Comment:
We don't need the source chain and destination chain in the EVM staking app, @devpavan04.